Dockerfile-lint

```diff
@@ -0,0 +1,7 @@
+# This dockerfile gives you a container with pylint, a development package installed.
```

Comment:
Can you rename this file to lint.Dockerfile?
https://stackoverflow.com/a/26084010